dist/chart/templates/rbac/role.yaml

```diff
@@ -0,0 +1,128 @@
+{{- if .Values.rbac.enable }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```

Comment:
Similar to the other comment, we may have updates for these roles in near future, is there a way to regenerate this file?

Reply:
Same as for the other comment.